Carmu/Sources/Extensions/UIViewController+Extension.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIViewController+Extension.swift
+//  Carmu
+//
+//  Created by 김태형 on 2023/11/05.
+//
+
+import UIKit
+
+extension UIViewController {
+
+    // TODO: - Firebase 형식에 맞게 변경
+    /// 운전자인지 여부 확인
+    func isCaptain() -> Bool {
+        if crewData?.captainID == "ted" {
+            return true
+        } else {
+            return false
+        }
+    }
```

Comment:
요렇게 쓰는게 더 깔끔할 것 같아요!

```suggestion
    func isCaptain() -> Bool {
        crewData?.captainID == "ted"
    }
```

Reply:
아니면 요런 방법도 있고요!

```suggestion
    var isCaptain: Bool {
        crewData?.captainID == "ted"
    }
```

Reply:
오호 참고하겠슴당 ㅎㅎ!